apps/gitness/src/pages/run-pipeline-dialog/run-pipeline-form.tsx

```diff
@@ -52,28 +63,41 @@ export default function RunPipelineForm({
 
   useEffect(() => {
     findPipeline({ pipeline_identifier: pipelineId ?? pipelineIdFromParams, repo_ref: repoRef })
-      .then(({ body: pipelineData }) => {
-        getContent({
-          path: pipelineData?.config_path ?? '',
-          repo_ref: repoRef,
-          queryParams: {
-            git_ref: normalizeGitRef(branch ?? pipelineData?.default_branch) ?? '',
-            include_commit: true
-          }
-        }).then(({ body: pipelineFileContent }) => {
-          try {
-            const pipelineObj = parse(decodeGitContent(pipelineFileContent?.content?.data))
-            setPipeline(pipelineObj)
-          } catch (ex) {
-            // TODO: toast
-            console.error(ex)
-          }
-          setLoading(false)
-        })
-      })
-      .catch(ex => {
-        // TODO: toast
-        console.log(ex)
+      .then(
+        ({ body: pipelineData }) => {
+          getContent({
+            path: pipelineData?.config_path ?? '',
+            repo_ref: repoRef,
+            queryParams: {
+              git_ref: normalizeGitRef(branch ?? pipelineData?.default_branch) ?? '',
+              include_commit: true
+            }
+          })
+            .then(
+              ({ body: pipelineFileContent }) => {
+                try {
+                  const pipelineObj = parse(decodeGitContent(pipelineFileContent?.content?.data))
+                  setPipeline(pipelineObj)
+                } catch (ex: unknown) {
+                  console.error(ex)
+                  setErrorMessage((ex as UsererrorError)?.message || null)
+                }
+              },
+              ex => {
+                console.error(ex)
```

Comment:
NIT: remove the console.errors if we are handling them with a dialog?

Reply:
Removed console error logs